openapi-spec.yaml

```diff
@@ -0,0 +1,846 @@
+openapi: "3.0.3"
+info:
+  title: "Backend Routes - Finternet"
```

Comment:
Backend at this level can mean Application backend. 
We are adding few more Ledger specific APIs as ledger APIs. That way, we can map them in implementation level.